JS/script.js

```diff
@@ -14,4 +14,39 @@ function generateCanvas(row, column) {
 
     document.querySelector('.canvas').innerHTML = canvas
 
-}
\ No newline at end of file
+}
+
+document.querySelector('html').addEventListener('mousedown', function () {
```

Comment:
using fat arrow functions and anonymous functions, please keep it consistent and stick to one, unless there is reason to use a particular one.

also anon functions should be:
```
function() {}
```
no space after the word function before the brackets